docs/DafnyRef/Statements.md

```diff
@@ -1004,6 +1033,13 @@ If the guard is an asterisk then a non-deterministic choice is made:
   }
 ```
 
+The then alternative of the if-statement must be block statement;
+the else alternative may be either a block statement or another if statement.
+The condition of the if statement need not (but may) be enclosed in parentheses.
+
+An if-statement with a binding guard is a ghost statement.
```

Comment:
Not true:
```dafny
method Test(i: int) returns (j: int) decreases i  {
  j := 1;
  if x :| 0 <= x < i < 10 {
    j := Test(x);
    j := j + 1;
  }
}
```
is not throwing any error. But the IDE highlights it  sometimes incorrectly as ghost

Reply:
OK - fixed.